src/foam/nanos/auth/PublicUserEntity.js

```diff
@@ -0,0 +1,73 @@
+foam.CLASS({
```

Comment:
Is this really a generic nanos feature? Would it be better to move to NP?

Reply:
I realized after opening this PR that it's only generic to foam so long as the subset of User properties comes from the User model defined in foam and doesn't include any refinements NANOPAY makes to User. Right now all of the properties come from the foam definition, so it could work as a generic nanos feature as is. Let me know if you'd like me to move it anyway though.

Reply:
Update: I have to move it because I'm going to have to add some properties to it that come from NANOPAY's refinement of User, namely `businessAddress` and `businessPhone`.